### ISSUE TITLE
Extended dataset

### ISSUE THREAD

**Issue by xiaoqian-shen** [ubc-vision/Make-A-Story#2]
Great work! I am curious about whether you will release the extended dataset with referential text.

**Comment by snarkyidiot** [comment]
I would also appreciate the release of this dataset

**Comment by xiaoqian-shen** [comment]
Btw, could you provide the pretrained character classifier for convenient evaluation?

**Comment by trahman8** [comment]
We re-used existing datasets. In the dataloader, we modify the ref texts. See ldm/data/flintstones_data.py, ldm/data/mugen_data.py and ldm/data/proro_data.py to generate ref text.

**Comment by xiaoqian-shen** [comment]
Thanks for your reply. This issue then could be closed.

**Comment by xiaoqian-shen** [comment]
Hi! I am trying to evaluate the accuracy of generated stories. Could you provide the pretrained weight of character and background classifier for convenient evaluation?
